### DISCUSSION TITLE
Can you please share a tutorial how build a spancat model above spacy blank model?

### DISCUSSION THREAD

**Discussion by steve-solun** [explosion/spaCy#11518] [Help: Best practices]
Please post an A-Z tutorial how can I build a spancat model given a spacy blank model (say "en") and DocBin object with my entities and labels.
I have read the docs, this repo and answers of the maintainers but still straggle to build it properly, please provide a simple working example to show how it should work.

**Comment by ljvmiranda921** [comment]
You can take a look at our sample projects to see end-to-end implementation of spancat (from preparing `DocBin` objects to training a model). I suggest looking at the [`ner_spancat`](https://github.com/explosion/projects/tree/v3/experimental/ner_spancat) project or [`spancat_food_ingredients`](https://github.com/explosion/projects/tree/v3/tutorials/spancat_food_ingredients) as a starting point. 

This may not be related to spancat, but I highly suggest reading through the [healthsea blogpost](https://explosion.ai/blog/healthsea) and checking out its [corresponding project](https://github.com/explosion/healthsea) to get a good sense of how to train spaCy components to your NLP problems.
